### PR TITLE
Small improvements for Tracking DQM

### PR DESCRIPTION
**Pull request by VinInn**
Some retune of histograms limits
new histogram of fraction of seeds becoming candidate

### CODE REVIEW

**Comment by VinInn** [comment]
@cmsbuild, please test

**Comment by VinInn** [comment]
the "error" in 10824.0 step5 is most probably not specifically due to this PR.
In addition there is no clear indication of what kind of error occurred and where....

**Comment by VinInn** [comment]
@cmsbuild , please test

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

File: DQM/TrackingMonitorSource/python/IterTrackingModules4seedMonitoring_cfi.py [hunk 1]
```diff
@@ -96,44 +100,48 @@
...
+clusterBin       ['muonSeededStepOutInDisplaced'] = cms.int32(100)
 clusterMax       ['muonSeededStepOutInDisplaced'] = cms.double(100000)
 
 seedInputTag     ['jetCoreRegionalStep'] = cms.InputTag("jetCoreRegionalStepSeeds")
 trackCandInputTag['jetCoreRegionalStep'] = cms.InputTag("jetCoreRegionalStepTrackCandidates")
-trackSeedSizeBin ['jetCoreRegionalStep'] = cms.int32(15)
+trackSeedSizeBin ['jetCoreRegionalStep'] = cms.int32(100)
 trackSeedSizeMin ['jetCoreRegionalStep'] = cms.double(-0.5)                         
-trackSeedSizeMax ['jetCoreRegionalStep'] = cms.double(14.5)
+trackSeedSizeMax ['jetCoreRegionalStep'] = cms.double(199.5)
```
Comment (VinInn) [inline DQM/TrackingMonitorSource/python/IterTrackingModules4seedMonitoring_cfi.py]: @slava77

Comment (slava77) [reply DQM/TrackingMonitorSource/python/IterTrackingModules4seedMonitoring_cfi.py]: +1
but is this enough ?
especially for a typical JetHT PD 2017 run

**Pull request merged**